metals/src/main/scala/scala/meta/internal/tvp/MetalsTreeViewProvider.scala

```diff
@@ -381,18 +404,34 @@ class FolderTreeViewProvider(
   def revealResult(
       path: AbsolutePath,
       closestSymbol: SymbolOccurrence,
```

Comment:
This seems to be offset by a line for me.
```
  def isNone: Boolean = value.isNone
  def is@@RootPackage: Boolean = value.isRootPackage
  ```
Reveals `isNone` in tree view.

Reply:
Yep, should be fixed and actually one of the tests was pointing to File class instead of object which were one after the other.